### DISCUSSION TITLE
Are there any working examples with harbor specified in REGISTRY_PROXY_REMOTEDOMAIN?

### DISCUSSION THREAD

**Discussion by kahirokunn** [distribution/distribution#4340] [Q&A] [answered]
I did the following but it didn't work with docker pull 127.0.0.1:5000/ubuntu .
I would like to do something about this.

```docker-compose.yml
services:
  harbor-project:
    image: registry:2
    volumes:
      - ./data/harbor-project:/var/lib/registry
    environment:
      REGISTRY_PROXY_REMOTEURL: https://my-harbor/project
    restart: always
    ports:
      - 5000:5000
```
I want you to send me stuff to `/v2/project`, but it gets sent to /v2/.
I feel like path is not respected.

**Comment by milosgajdos** [comment, marked as answer]
That's likely because the `/v2` path is hardcoded in the proxy:

https://github.com/distribution/distribution/blob/c49220d492233615f22c1eea91e3121a681751eb/registry/proxy/proxyregistry.go#L242

**Reply by kahirokunn** [reply]
Thank you!
I am looking to create a PR and support Harbor.

https://github.com/distribution/distribution/pull/4342
